### ISSUE TITLE
Can we enable pitch extension now?

### ISSUE THREAD

**Issue by chrissearle** [dbaldwin/DronePan#53]
Code in GimbalController - can we use this to enable pitch up? On all devices?

```
        if let rangeExtension = gimbal.gimbalCapability[DJIGimbalKeyPitchRangeExtension] as? DJIParamCapability {
            DDLogDebug("Range extension supported: \(rangeExtension.isSupported)")

            // TOOD - we should now be able to check and set sky row. Testing on all devices needed.
            /*
            if rangeExtension.isSupported {
                gimbal!.setPitchRangeExtensionEnabled(true, withCompletion: nil)
            }
            */
        }
```

**Comment by chrissearle** [comment]
Once we push out b5 - want some logs from P3 and P4 users - what does it say in the logs under "Range extension supported".

**Comment by chrissearle** [comment]
Once we have the new settings setup in place #74 #72 then this will be easier to implement - an "if available then enable and store in settings" and we should be good to go.

**Comment by chrissearle** [comment]
Implemented in 1.5
